app/assets/javascripts/mobile/mobile_comments.js

```diff
@@ -11,6 +11,8 @@
     initialize: function() {
       var self = this;
 
+      new Diaspora.MarkdownEditor(".comment_box");
```

Comment:
Ideally, you'd have the constructor doing nothing besides internal state. There are multiple reasons for that, but mainly there are very valid reasons to have the render function called again under some circumstances, or to build objects early when the DOM is not even finishing loading, dispatching rendering to a later point in time to increase the apparent performance.

I agree that this is something we could change, but as @svbergerem said, let's not do that here.